actionpack-action_caching.gemspec

```
@@ -14,8 +14,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license       = "MIT"
 
-  gem.add_dependency "actionpack", ">= 4.0.0", "< 6"
+  gem.add_dependency "actionpack", ">= 4.0.0", "< 7"
```

Can you remove the upper limit?

```suggestion
  gem.add_dependency "actionpack", ">= 4.0.0"
```